Tasks/AzureResourceGroupDeploymentV2/operations/ResourceGroup.ts

```diff
@@ -180,6 +193,45 @@ export class ResourceGroup {
         }
     }
 
+    private getPolicyHelpLink(errorDetailMessage): string {
+        let managementGroupId = this.parseManagementGroupId(errorDetailMessage);
+        let portalUrl = this.taskParameters.endpointPortalUrl ? this.taskParameters.endpointPortalUrl : "https://portal.azure.com";
+        let policyLink = portalUrl + "#blade/Microsoft_Azure_Policy/PolicyMenuBlade/Assignments/scope/%2Fproviders%2FMicrosoft.Management%2FmanagementGroups%2F" + managementGroupId;
```

Comment:
Make sure this URL works, also if the scope is as expected.

Reply:
@ds-ms  validated. value next to `/scope/` represents the actual scope of policy `%2Fproviders%2FMicrosoft.Management%2FmanagementGroups%2FAjayMG`